modules/classes.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
`Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice. Instead of disabling `eslint`, it will be nice to fix the issue. This will even make your code more modular which is a best practice.